shared/src/common/enums/validation/master/master-sign-up.validation-message.ts

```diff
@@ -1,6 +1,13 @@
+import { MasterSignUpValidationRule } from './master-sign-up.validation-rule';
```

Comment:
rename file to `shared/src/common/enums/validation/master/master-sign-up-validation-message.enum.ts`